graphics/MoltenVK/Portfile

```diff
@@ -3,14 +3,14 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        KhronosGroup MoltenVK 1.1.9 v
+github.setup        KhronosGroup MoltenVK 1.2.4 v
+github.tarball_from releases
 epoch               1
```

Comment:
No need to set epoch for the new subports.

Reply:
I’ve removed the epoch from here but retained it for the stub as I’d changed the versioning for the stub